src/plugins/vis_augmenter/public/vega/constants.ts

```diff
@@ -0,0 +1,12 @@
+/*
+ * Copyright OpenSearch Contributors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+export enum VisInteraction {
+  VIEW_EVENTS_FLYOUT = 'viewEventsFlyout',
+}
+
+export enum VisInteractionHandler {
+  HANDLE_POINT_IN_TIME_CLICK = 'visAugmenter.handlePointInTimeClick',
```

Comment:
nit: suggest to include `EVENT` somewhere in the name so it helps differentiate it and prevent confusion of someone thinking this includes other points on the chart, like the regular generated points from the metrics, etc.

Something like `HANDLE_POINT_IN_TIME_EVENT_CLICK` or `HANDLE_EVENT_CLICK`?

Reply:
Changing to `VisInteractionEventHandlerName`

Reply:
The point here was to change the actual enum values to have `EVENT` in the naming, to help clarify that the clicking is happening on a point in time event, rather than an arbitrary datapoint within a point-in-time-enabled chart.

Reply:
Done